src/viewModelBuilders/common/utils.ts

```diff
@@ -19,27 +19,27 @@ export function mapCategoryKeyLabel(
 }
 
 /**
- * Sanitizes a string for display i.e. any empty, null or undefined value is sanitized to "Unspecified".
- * @param str - String to sanitize.
+ * Sanitizes a value to a string for display i.e. any empty, null or undefined value is sanitized to "Unspecified".
+ * @param value - Value to sanitize.
  * @returns the string or sanitized string value.
  */
-export function sanitizeString(str: string): string {
-  if (str === "" || str === null || str === undefined) {
+export function sanitizeString(value: unknown): string {
```

Comment:
I found that there were type errors from `sanitizeString` when I tried using the package with `data-browser`, so I updated it to not just take strings (which I suppose is probably how it should've been typed anyway if it's checking for `null` and `undefined`)